tests/pipeline/test_filter.py

```diff
@@ -820,3 +822,40 @@ def test_top_and_bottom_with_groupby_and_mask(self, dtype, seed):
             },
             mask=self.build_mask(permute(rot90(self.eye_mask(shape=shape)))),
         )
+
+
+class SpecificAssetsTestCase(WithSeededRandomPipelineEngine,
```

Comment:
We could, but setting up the engine with sids and trading days is a little bit boilerplate-ey.  I tend to use `WithSeededRandomPipelineEngine` when I just need **an** engine, but I don't care about the specific values it emits.